From 0c729b5d863e2f3723cfb5c3bdef7ecf6c066b5e Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 8 Jul 2024 10:58:32 -0700
Subject: [PATCH] Drop support for ocaml autoprov/dep

There aren't enough ocaml packages in the distro and this is causing
problems with the latest release to bother fixing up.

Signed-off-by: William Douglas <william.douglas@intel.com>
---
 fileattrs/ocaml.attr | 2 --
 1 file changed, 2 deletions(-)

diff --git a/fileattrs/ocaml.attr b/fileattrs/ocaml.attr
index 20f0529..36401e9 100644
--- a/fileattrs/ocaml.attr
+++ b/fileattrs/ocaml.attr
@@ -1,5 +1,3 @@
-%__ocaml_provides	%{_rpmconfigdir}/ocamldeps.sh --provides
-%__ocaml_requires	%{_rpmconfigdir}/ocamldeps.sh --requires
 %__ocaml_magic		^(Objective caml|OCaml) .*$
 %__ocaml_path		.(cma|cmi|cmo|cmx|cmxa)$
 %__ocaml_flags		magic_and_path
-- 
2.45.1

